Makefile

```diff
@@ -1,17 +1,24 @@
 # This file is for you! Edit it to implement your own hooks (make targets) into
 # the project as automated steps to be executed on locally and in the CD pipeline.
 
-include scripts/init.mk
+include ./scripts/init.mk
 
-# ==============================================================================
```

Comment:
Can we keep the "grouping" comment, please? We use that convention across all the make files and shell scripts in this repository.